From: zuozhiwei1 <2810236083@qq.com>
Date: Wed, 10 Jul 2024 09:34:26 +0000
Subject: Translated using Weblate (Kazakh)

Currently translated at 100.0% (73 of 73 strings)

Translation: openkylin-nile-new/ukui-volume-control-apply-qt
Translate-URL: http://weblate.openkylin.top/projects/openkylin-nile-new/ukui-volume-control-apply-qt/kk/
---
 ukui-volume-control-applet-qt/translations/kk.ts | 146 +++++++++++------------
 ukui-volume-control-applet-qt/translations/ky.ts | 146 +++++++++++------------
 2 files changed, 146 insertions(+), 146 deletions(-)

diff --git a/ukui-volume-control-applet-qt/translations/kk.ts b/ukui-volume-control-applet-qt/translations/kk.ts
index 533e303..19cbb1a 100644
--- a/ukui-volume-control-applet-qt/translations/kk.ts
+++ b/ukui-volume-control-applet-qt/translations/kk.ts
@@ -6,17 +6,17 @@
     <message>
         <location filename="../ukmedia_application_volume_widget.cpp" line="43"/>
         <source>Application Volume</source>
-        <translation type="unfinished"></translation>
+        <translation>Қолданба көлемі</translation>
     </message>
     <message>
         <location filename="../ukmedia_application_volume_widget.cpp" line="32"/>
         <source>System Volume</source>
-        <translation type="unfinished"></translation>
+        <translation>Жүйе көлемі</translation>
     </message>
     <message>
         <location filename="../ukmedia_application_volume_widget.cpp" line="96"/>
         <source>Sound Settings</source>
-        <translation type="unfinished"></translation>
+        <translation>Дыбыс параметрлері</translation>
     </message>
 </context>
 <context>
@@ -79,170 +79,170 @@
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="2182"/>
         <source>pa_context_subscribe() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_context_subscribe () жаңылысы</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="2205"/>
         <source>pa_context_get_card_info_list() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_context_get_card_info_list () жаңылысы</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="2212"/>
         <source>pa_context_get_sink_info_list() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_context_get_sink_info_list () жаңылысы</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="2219"/>
         <source>pa_context_get_source_info_list() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_context_get_source_info_list () жаңылысы</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="1936"/>
         <source>Failed to initialize stream_restore extension: %s</source>
-        <translation type="unfinished"></translation>
+        <translation>stream_restore кеңейтімі басталмады:% s</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="1951"/>
         <source>pa_ext_stream_restore_read() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_ext_stream_restore_read () жаңылысы</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="2044"/>
         <source>pa_context_get_sink_info_by_index() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_context_get_sink_info_by_index () жаңылысы</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="2057"/>
         <source>pa_context_get_source_info_by_index() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_context_get_source_info_by_index () жаңылысы</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="1920"/>
         <location filename="../ukmedia_volume_control.cpp" line="2118"/>
         <location filename="../ukmedia_volume_control.cpp" line="2144"/>
         <source>pa_context_get_card_info_by_index() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_context_get_card_info_by_index () жаңылысы</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="1557"/>
         <location filename="../ukmedia_volume_control.cpp" line="1576"/>
         <location filename="../ukmedia_volume_control.cpp" line="1606"/>
         <source>Card callback failure</source>
-        <translation type="unfinished"></translation>
+        <translation>Картаның қоңырау шалу сәтсіздігі</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="1640"/>
         <location filename="../ukmedia_volume_control.cpp" line="1756"/>
         <source>Sink callback failure</source>
-        <translation type="unfinished"></translation>
+        <translation>Қоңырау шалу сәтсіздігі</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="1698"/>
         <location filename="../ukmedia_volume_control.cpp" line="1778"/>
         <source>Source callback failure</source>
-        <translation type="unfinished"></translation>
+        <translation>Бастапқы қоңырау шалу сәтсіздігі</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="97"/>
         <source>Fatal Error: Unable to connect to PulseAudio</source>
-        <translation type="unfinished"></translation>
+        <translation>Өлім қатесі: PulseAudio-ға қосыла алмады</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="287"/>
         <location filename="../ukmedia_volume_control.cpp" line="2106"/>
         <location filename="../ukmedia_volume_control.cpp" line="2191"/>
         <source>pa_context_get_server_info() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_context_get_server_info () жаңылысы</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="1799"/>
         <location filename="../ukmedia_volume_control.cpp" line="2336"/>
         <source>Sink input callback failure</source>
-        <translation type="unfinished"></translation>
+        <translation>Синквейн енгізу қоңырауының істен шығуы</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="1818"/>
         <source>Source output callback failure</source>
-        <translation type="unfinished"></translation>
+        <translation>Бастапқы шығыс шақырудың істен шығуы</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="1846"/>
         <source>Client callback failure</source>
-        <translation type="unfinished"></translation>
+        <translation>Клиенттің қоңырау шалу сәтсіздігі</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="1862"/>
         <location filename="../ukmedia_volume_control.cpp" line="1906"/>
         <source>Server info callback failure</source>
-        <translation type="unfinished"></translation>
+        <translation>Сервер ақпаратының қоңырау шалу қатесі</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="1969"/>
         <source>Failed to initialize device restore extension: %s</source>
-        <translation type="unfinished"></translation>
+        <translation>Құрылғыны қалпына келтіру жаңылысы:% s</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="1990"/>
         <source>pa_ext_device_restore_read_sink_formats() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_ext_device_restore_read_sink_formats () жаңылысы</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="2008"/>
         <source>Failed to initialize device manager extension: %s</source>
-        <translation type="unfinished"></translation>
+        <translation>Құрылғы менеджерінің кеңейтімі басталмады:% s</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="2027"/>
         <source>pa_ext_device_manager_read() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_ext_device_manager_read () жаңылысы</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="2070"/>
         <location filename="../ukmedia_volume_control.cpp" line="2083"/>
         <source>pa_context_get_sink_input_info() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_context_get_sink_input_info () жаңылысы</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="2096"/>
         <source>pa_context_get_client_info() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_context_get_client_info () жаңылысы</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="2198"/>
         <source>pa_context_client_info_list() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_context_client_info_list () жаңылысы</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="2226"/>
         <source>pa_context_get_sink_input_info_list() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_context_get_sink_input_info_list () жаңылысы</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="2233"/>
         <source>pa_context_get_source_output_info_list() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_context_get_source_output_info_list () жаңылысы</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="2267"/>
         <source>moduleInfoCb callback failure</source>
-        <translation type="unfinished"></translation>
+        <translation>moduleInfoCb қоңырау шалу сәтсіздігі</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="2284"/>
         <source>Ukui Media Volume Control</source>
-        <translation type="unfinished"></translation>
+        <translation>Ukui Media көлемін бақылау</translation>
     </message>
     <message>
         <location filename="../ukmedia_main_widget.cpp" line="3044"/>
         <location filename="../ukmedia_main_widget.cpp" line="3071"/>
         <location filename="../ukmedia_main_widget.cpp" line="3126"/>
         <source>pa_context_load_module() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_context_load_module () жаңылысы</translation>
     </message>
     <message>
         <location filename="../main.cpp" line="72"/>
         <source>ukui-volume-control-applet-qt is already running!</source>
-        <translation type="unfinished"></translation>
+        <translation>ukui-volume-control-applet-qt қазірдің өзінде іске қосылған!</translation>
     </message>
 </context>
 <context>
@@ -273,7 +273,7 @@
     <message>
         <location filename="../ukmedia_main_widget.cpp" line="110"/>
         <source>Output volume control</source>
-        <translation type="unfinished"></translation>
+        <translation>Шығарылым көлемін бақылау</translation>
     </message>
     <message>
         <source>Mute</source>
@@ -283,22 +283,22 @@
         <location filename="../ukmedia_main_widget.cpp" line="112"/>
         <location filename="../ukmedia_main_widget.cpp" line="114"/>
         <source>Sound preference(S)</source>
-        <translation type="unfinished"></translation>
+        <translation>Дыбыстық артықшылық(S)</translation>
     </message>
     <message>
         <location filename="../ukmedia_main_widget.cpp" line="180"/>
         <source>System Volume</source>
-        <translation type="unfinished"></translation>
+        <translation>Жүйе көлемі</translation>
     </message>
     <message>
         <location filename="../ukmedia_main_widget.cpp" line="182"/>
         <source>App Volume</source>
-        <translation type="unfinished"></translation>
+        <translation>Бағдарлама көлемі</translation>
     </message>
     <message>
         <location filename="../ukmedia_main_widget.cpp" line="788"/>
         <source>Current volume:</source>
-        <translation type="unfinished"></translation>
+        <translation>Ағымдағы көлем:</translation>
     </message>
 </context>
 <context>
@@ -317,17 +317,17 @@
     <message>
         <location filename="../ukmedia_system_volume_widget.cpp" line="74"/>
         <source>Volume</source>
-        <translation type="unfinished"></translation>
+        <translation>Көлемі</translation>
     </message>
     <message>
         <location filename="../ukmedia_system_volume_widget.cpp" line="86"/>
         <source>Output</source>
-        <translation type="unfinished"></translation>
+        <translation>Шығыс</translation>
     </message>
     <message>
         <location filename="../ukmedia_system_volume_widget.cpp" line="103"/>
         <source>Sound Settings</source>
-        <translation type="unfinished"></translation>
+        <translation>Дыбыс параметрлері</translation>
     </message>
 </context>
 <context>
@@ -336,7 +336,7 @@
         <location filename="../ukmedia_volume_control.cpp" line="108"/>
         <location filename="../ukmedia_volume_control.cpp" line="129"/>
         <source>pa_context_set_sink_mute_by_index() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_context_set_sink_mute_by_index () жаңылысы</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="134"/>
@@ -345,128 +345,128 @@
         <location filename="../ukmedia_volume_control.cpp" line="343"/>
         <location filename="../ukmedia_volume_control.cpp" line="360"/>
         <source>pa_context_set_sink_volume_by_index() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_context_set_sink_volume_by_index () жаңылысы</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="154"/>
         <location filename="../ukmedia_volume_control.cpp" line="211"/>
         <source>pa_context_set_source_mute_by_index() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_context_set_source_mute_by_index () жаңылысы</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="159"/>
         <location filename="../ukmedia_volume_control.cpp" line="242"/>
         <source>pa_context_set_source_volume_by_index() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_context_set_source_volume_by_index () жаңылысы</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="380"/>
         <source>pa_context_move_sink_input_by_index() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_context_move_sink_input_by_index () жаңылысы</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="399"/>
         <source>pa_context_move_source_output_by_name() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_context_move_source_output_by_name () жаңылысы</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="420"/>
         <location filename="../ukmedia_volume_control.cpp" line="424"/>
         <source>pa_context_set_source_output_volume() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_context_set_source_output_volume () жаңылысы</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="442"/>
         <source>pa_context_set_source_output_mute() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_context_set_source_output_mute () жаңылысы</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="460"/>
         <source>pa_context_set_card_profile_by_index() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_context_set_card_profile_by_index () жаңылысы</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="479"/>
         <location filename="../ukmedia_volume_control.cpp" line="486"/>
         <source>pa_context_set_default_sink() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_context_set_default_sink () жаңылысы</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="504"/>
         <source>pa_context_set_default_source() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_context_set_default_source () жаңылысы</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="520"/>
         <source>pa_context_set_sink_port_by_name() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_context_set_sink_port_by_name () жаңылысы</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="535"/>
         <source>pa_context_set_source_port_by_name() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_context_set_source_port_by_name () жаңылысы</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="548"/>
         <source>pa_context_kill_sink_input() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_context_kill_sink_input () жаңылысы</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="577"/>
         <source> (plugged in)</source>
-        <translation type="unfinished"></translation>
+        <translation> (жалғанған)</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="581"/>
         <location filename="../ukmedia_volume_control.cpp" line="739"/>
         <source> (unavailable)</source>
-        <translation type="unfinished"></translation>
+        <translation> (қол жетімді емес)</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="583"/>
         <location filename="../ukmedia_volume_control.cpp" line="736"/>
         <source> (unplugged)</source>
-        <translation type="unfinished"></translation>
+        <translation> (ажыратылған)</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="1072"/>
         <source>Failed to read data from stream</source>
-        <translation type="unfinished"></translation>
+        <translation>Ағыннан деректер оқылмады</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="1113"/>
         <source>Peak detect</source>
-        <translation type="unfinished"></translation>
+        <translation>Шыңды анықтау</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="1114"/>
         <source>Failed to create monitoring stream</source>
-        <translation type="unfinished"></translation>
+        <translation>Бақылау ағынын жасау жаңылысы</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="1129"/>
         <source>Failed to connect monitoring stream</source>
-        <translation type="unfinished"></translation>
+        <translation>Бақылау ағыны қосылмады</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="1290"/>
         <source>Ignoring sink-input due to it being designated as an event and thus handled by the Event widget</source>
-        <translation type="unfinished"></translation>
+        <translation>Оқиға ретінде белгіленуіне және осылайша Оқиға виджетінің өңдеуіне байланысты раковина енгізуді елемеу</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="1527"/>
         <source>Establishing connection to PulseAudio. Please wait...</source>
-        <translation type="unfinished"></translation>
+        <translation>PulseAudio-мен байланыс орнату. Күте тұршы...</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="1871"/>
         <source>pa_context_get_sink_info_by_name() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_context_get_sink_info_by_name () жаңылысы</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="1875"/>
         <source>pa_context_get_source_info_by_name() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_context_get_source_info_by_name () жаңылысы</translation>
     </message>
 </context>
 <context>
@@ -474,32 +474,32 @@
     <message>
         <location filename="../ukui_media_set_headset_widget.cpp" line="36"/>
         <source>Sound Settings</source>
-        <translation type="unfinished"></translation>
+        <translation>Дыбыс параметрлері</translation>
     </message>
     <message>
         <location filename="../ukui_media_set_headset_widget.cpp" line="37"/>
         <source>Cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>Болдырмау</translation>
     </message>
     <message>
         <location filename="../ukui_media_set_headset_widget.cpp" line="38"/>
         <source>Select Sound Device</source>
-        <translation type="unfinished"></translation>
+        <translation>Дыбыс құрылғысын таңдау</translation>
     </message>
     <message>
         <location filename="../ukui_media_set_headset_widget.cpp" line="67"/>
         <source>Headphone</source>
-        <translation type="unfinished"></translation>
+        <translation>Құлаққап</translation>
     </message>
     <message>
         <location filename="../ukui_media_set_headset_widget.cpp" line="68"/>
         <source>Headset</source>
-        <translation type="unfinished"></translation>
+        <translation>Гарнитура</translation>
     </message>
     <message>
         <location filename="../ukui_media_set_headset_widget.cpp" line="69"/>
         <source>Microphone</source>
-        <translation type="unfinished"></translation>
+        <translation>Микрофон</translation>
     </message>
 </context>
 </TS>
diff --git a/ukui-volume-control-applet-qt/translations/ky.ts b/ukui-volume-control-applet-qt/translations/ky.ts
index 4292a3f..124b8ba 100644
--- a/ukui-volume-control-applet-qt/translations/ky.ts
+++ b/ukui-volume-control-applet-qt/translations/ky.ts
@@ -6,17 +6,17 @@
     <message>
         <location filename="../ukmedia_application_volume_widget.cpp" line="43"/>
         <source>Application Volume</source>
-        <translation type="unfinished"></translation>
+        <translation>Иштеменин көлөмү</translation>
     </message>
     <message>
         <location filename="../ukmedia_application_volume_widget.cpp" line="32"/>
         <source>System Volume</source>
-        <translation type="unfinished"></translation>
+        <translation>Системанын көлөмү</translation>
     </message>
     <message>
         <location filename="../ukmedia_application_volume_widget.cpp" line="96"/>
         <source>Sound Settings</source>
-        <translation type="unfinished"></translation>
+        <translation>Үн параметрлери</translation>
     </message>
 </context>
 <context>
@@ -79,170 +79,170 @@
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="2182"/>
         <source>pa_context_subscribe() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_context_subscribe() ишке ашпады</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="2205"/>
         <source>pa_context_get_card_info_list() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_context_get_card_info_list() ишке ашпады</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="2212"/>
         <source>pa_context_get_sink_info_list() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_context_get_sink_info_list() ишке ашпады</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="2219"/>
         <source>pa_context_get_source_info_list() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_context_get_source_info_list() ишке ашпады</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="1936"/>
         <source>Failed to initialize stream_restore extension: %s</source>
-        <translation type="unfinished"></translation>
+        <translation>Кеңейтүү stream_restore иниализациялоо ишке ашпады: %s</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="1951"/>
         <source>pa_ext_stream_restore_read() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_ext_stream_restore_read() ишке ашпады</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="2044"/>
         <source>pa_context_get_sink_info_by_index() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_context_get_sink_info_by_index() ишке ашпады</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="2057"/>
         <source>pa_context_get_source_info_by_index() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_context_get_source_info_by_index() ишке ашпады</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="1920"/>
         <location filename="../ukmedia_volume_control.cpp" line="2118"/>
         <location filename="../ukmedia_volume_control.cpp" line="2144"/>
         <source>pa_context_get_card_info_by_index() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_context_get_card_info_by_index() ишке ашпады</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="1557"/>
         <location filename="../ukmedia_volume_control.cpp" line="1576"/>
         <location filename="../ukmedia_volume_control.cpp" line="1606"/>
         <source>Card callback failure</source>
-        <translation type="unfinished"></translation>
+        <translation>Картаны кайтаруу катасы</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="1640"/>
         <location filename="../ukmedia_volume_control.cpp" line="1756"/>
         <source>Sink callback failure</source>
-        <translation type="unfinished"></translation>
+        <translation>Батып кетүү катасы</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="1698"/>
         <location filename="../ukmedia_volume_control.cpp" line="1778"/>
         <source>Source callback failure</source>
-        <translation type="unfinished"></translation>
+        <translation>Булакка чалуу катасы</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="97"/>
         <source>Fatal Error: Unable to connect to PulseAudio</source>
-        <translation type="unfinished"></translation>
+        <translation>Өлүмгө дуушар болгон ката: PulseAudio-га туташууга мүмкүн эмес</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="287"/>
         <location filename="../ukmedia_volume_control.cpp" line="2106"/>
         <location filename="../ukmedia_volume_control.cpp" line="2191"/>
         <source>pa_context_get_server_info() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_context_get_server_info() ишке ашпады</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="1799"/>
         <location filename="../ukmedia_volume_control.cpp" line="2336"/>
         <source>Sink input callback failure</source>
-        <translation type="unfinished"></translation>
+        <translation>Батып кирүү чакырыгынын жетишсиздиги</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="1818"/>
         <source>Source output callback failure</source>
-        <translation type="unfinished"></translation>
+        <translation>Булак чыгаруу чакыруу катасы</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="1846"/>
         <source>Client callback failure</source>
-        <translation type="unfinished"></translation>
+        <translation>Кардарларды кайтаруу катасы</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="1862"/>
         <location filename="../ukmedia_volume_control.cpp" line="1906"/>
         <source>Server info callback failure</source>
-        <translation type="unfinished"></translation>
+        <translation>Сервердин маалыматы кайрымдуулук ишсиздиги</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="1969"/>
         <source>Failed to initialize device restore extension: %s</source>
-        <translation type="unfinished"></translation>
+        <translation>Кеңейтүүнү калыбына келтирүү үчүн аппаратты иниализациялоо ишке ашпады: %s</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="1990"/>
         <source>pa_ext_device_restore_read_sink_formats() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_ext_device_restore_read_sink_formats() ишке ашпады</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="2008"/>
         <source>Failed to initialize device manager extension: %s</source>
-        <translation type="unfinished"></translation>
+        <translation>Түзмөк менеджерин кеңейтүүнү иниализациялоо ишке ашпады: %s</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="2027"/>
         <source>pa_ext_device_manager_read() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_ext_device_manager_read() ишке ашпады</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="2070"/>
         <location filename="../ukmedia_volume_control.cpp" line="2083"/>
         <source>pa_context_get_sink_input_info() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_context_get_sink_input_info() ишке ашпады</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="2096"/>
         <source>pa_context_get_client_info() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_context_get_client_info() ишке ашпады</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="2198"/>
         <source>pa_context_client_info_list() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_context_client_info_list() ишке ашпады</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="2226"/>
         <source>pa_context_get_sink_input_info_list() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_context_get_sink_input_info_list() ишке ашпады</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="2233"/>
         <source>pa_context_get_source_output_info_list() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_context_get_source_output_info_list() ишке ашпады</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="2267"/>
         <source>moduleInfoCb callback failure</source>
-        <translation type="unfinished"></translation>
+        <translation>moduleInfoCb чалуу катасы</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="2284"/>
         <source>Ukui Media Volume Control</source>
-        <translation type="unfinished"></translation>
+        <translation>Укуи медиа көлөмүн көзөмөлдөө</translation>
     </message>
     <message>
         <location filename="../ukmedia_main_widget.cpp" line="3044"/>
         <location filename="../ukmedia_main_widget.cpp" line="3071"/>
         <location filename="../ukmedia_main_widget.cpp" line="3126"/>
         <source>pa_context_load_module() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_context_load_module() ишке ашпады</translation>
     </message>
     <message>
         <location filename="../main.cpp" line="72"/>
         <source>ukui-volume-control-applet-qt is already running!</source>
-        <translation type="unfinished"></translation>
+        <translation>ukui-том-башкаруу-апплет-qt буга чейин иштеп жатат!</translation>
     </message>
 </context>
 <context>
@@ -273,7 +273,7 @@
     <message>
         <location filename="../ukmedia_main_widget.cpp" line="110"/>
         <source>Output volume control</source>
-        <translation type="unfinished"></translation>
+        <translation>Чыгаруу көлөмүн көзөмөлдөө</translation>
     </message>
     <message>
         <source>Mute</source>
@@ -283,22 +283,22 @@
         <location filename="../ukmedia_main_widget.cpp" line="112"/>
         <location filename="../ukmedia_main_widget.cpp" line="114"/>
         <source>Sound preference(S)</source>
-        <translation type="unfinished"></translation>
+        <translation>Туура артыкчылык (S)</translation>
     </message>
     <message>
         <location filename="../ukmedia_main_widget.cpp" line="180"/>
         <source>System Volume</source>
-        <translation type="unfinished"></translation>
+        <translation>Системанын көлөмү</translation>
     </message>
     <message>
         <location filename="../ukmedia_main_widget.cpp" line="182"/>
         <source>App Volume</source>
-        <translation type="unfinished"></translation>
+        <translation>Тиркеменин көлөмү</translation>
     </message>
     <message>
         <location filename="../ukmedia_main_widget.cpp" line="788"/>
         <source>Current volume:</source>
-        <translation type="unfinished"></translation>
+        <translation>Учурдагы көлөмү:</translation>
     </message>
 </context>
 <context>
@@ -317,17 +317,17 @@
     <message>
         <location filename="../ukmedia_system_volume_widget.cpp" line="74"/>
         <source>Volume</source>
-        <translation type="unfinished"></translation>
+        <translation>Том</translation>
     </message>
     <message>
         <location filename="../ukmedia_system_volume_widget.cpp" line="86"/>
         <source>Output</source>
-        <translation type="unfinished"></translation>
+        <translation>Чыгаруу</translation>
     </message>
     <message>
         <location filename="../ukmedia_system_volume_widget.cpp" line="103"/>
         <source>Sound Settings</source>
-        <translation type="unfinished"></translation>
+        <translation>Үн параметрлери</translation>
     </message>
 </context>
 <context>
@@ -336,7 +336,7 @@
         <location filename="../ukmedia_volume_control.cpp" line="108"/>
         <location filename="../ukmedia_volume_control.cpp" line="129"/>
         <source>pa_context_set_sink_mute_by_index() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_context_set_sink_mute_by_index() ишке ашпады</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="134"/>
@@ -345,128 +345,128 @@
         <location filename="../ukmedia_volume_control.cpp" line="343"/>
         <location filename="../ukmedia_volume_control.cpp" line="360"/>
         <source>pa_context_set_sink_volume_by_index() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_context_set_sink_volume_by_index() ишке ашпады</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="154"/>
         <location filename="../ukmedia_volume_control.cpp" line="211"/>
         <source>pa_context_set_source_mute_by_index() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_context_set_source_mute_by_index() ишке ашпады</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="159"/>
         <location filename="../ukmedia_volume_control.cpp" line="242"/>
         <source>pa_context_set_source_volume_by_index() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_context_set_source_volume_by_index() ишке ашпады</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="380"/>
         <source>pa_context_move_sink_input_by_index() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_context_move_sink_input_by_index() ишке ашпады</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="399"/>
         <source>pa_context_move_source_output_by_name() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_context_move_source_output_by_name() ишке ашпады</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="420"/>
         <location filename="../ukmedia_volume_control.cpp" line="424"/>
         <source>pa_context_set_source_output_volume() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_context_set_source_output_volume() ишке ашпады</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="442"/>
         <source>pa_context_set_source_output_mute() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_context_set_source_output_mute() ишке ашпады</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="460"/>
         <source>pa_context_set_card_profile_by_index() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_context_set_card_profile_by_index() ишке ашпады</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="479"/>
         <location filename="../ukmedia_volume_control.cpp" line="486"/>
         <source>pa_context_set_default_sink() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_context_set_default_sink() ишке ашпады</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="504"/>
         <source>pa_context_set_default_source() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_context_set_default_source() ишке ашпады</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="520"/>
         <source>pa_context_set_sink_port_by_name() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_context_set_sink_port_by_name() ишке ашпады</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="535"/>
         <source>pa_context_set_source_port_by_name() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_context_set_source_port_by_name() ишке ашпады</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="548"/>
         <source>pa_context_kill_sink_input() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_context_kill_sink_input() ишке ашпады</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="577"/>
         <source> (plugged in)</source>
-        <translation type="unfinished"></translation>
+        <translation> (туташтырылган)</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="581"/>
         <location filename="../ukmedia_volume_control.cpp" line="739"/>
         <source> (unavailable)</source>
-        <translation type="unfinished"></translation>
+        <translation> (Жеткиликтүү эмес)</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="583"/>
         <location filename="../ukmedia_volume_control.cpp" line="736"/>
         <source> (unplugged)</source>
-        <translation type="unfinished"></translation>
+        <translation> (ажыратылбаган)</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="1072"/>
         <source>Failed to read data from stream</source>
-        <translation type="unfinished"></translation>
+        <translation>Агымдын маалыматтарын окуй алган жок</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="1113"/>
         <source>Peak detect</source>
-        <translation type="unfinished"></translation>
+        <translation>Эң жогорку аныктоо</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="1114"/>
         <source>Failed to create monitoring stream</source>
-        <translation type="unfinished"></translation>
+        <translation>Мониторинг агымын түзө алган жок</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="1129"/>
         <source>Failed to connect monitoring stream</source>
-        <translation type="unfinished"></translation>
+        <translation>Мониторинг агымын туташтыруу ишке ашпады</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="1290"/>
         <source>Ignoring sink-input due to it being designated as an event and thus handled by the Event widget</source>
-        <translation type="unfinished"></translation>
+        <translation>Иш-чара катары белгиленгендиктен жана иш-чаранын виджетинин иш-чарасы менен алектенгендиктен батып кетүүнү четке кагуу</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="1527"/>
         <source>Establishing connection to PulseAudio. Please wait...</source>
-        <translation type="unfinished"></translation>
+        <translation>ПульсеАудио менен байланыш түзүү. Сураныч күтө туруңуз...</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="1871"/>
         <source>pa_context_get_sink_info_by_name() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_context_get_sink_info_by_name() ишке ашпады</translation>
     </message>
     <message>
         <location filename="../ukmedia_volume_control.cpp" line="1875"/>
         <source>pa_context_get_source_info_by_name() failed</source>
-        <translation type="unfinished"></translation>
+        <translation>pa_context_get_source_info_by_name() ишке ашпады</translation>
     </message>
 </context>
 <context>
@@ -474,32 +474,32 @@
     <message>
         <location filename="../ukui_media_set_headset_widget.cpp" line="36"/>
         <source>Sound Settings</source>
-        <translation type="unfinished"></translation>
+        <translation>Үн параметрлери</translation>
     </message>
     <message>
         <location filename="../ukui_media_set_headset_widget.cpp" line="37"/>
         <source>Cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>Жокко чыгаруу</translation>
     </message>
     <message>
         <location filename="../ukui_media_set_headset_widget.cpp" line="38"/>
         <source>Select Sound Device</source>
-        <translation type="unfinished"></translation>
+        <translation>Үн түзмөктү тандоо</translation>
     </message>
     <message>
         <location filename="../ukui_media_set_headset_widget.cpp" line="67"/>
         <source>Headphone</source>
-        <translation type="unfinished"></translation>
+        <translation>Кулагы</translation>
     </message>
     <message>
         <location filename="../ukui_media_set_headset_widget.cpp" line="68"/>
         <source>Headset</source>
-        <translation type="unfinished"></translation>
+        <translation>Гарнитураны</translation>
     </message>
     <message>
         <location filename="../ukui_media_set_headset_widget.cpp" line="69"/>
         <source>Microphone</source>
-        <translation type="unfinished"></translation>
+        <translation>Микрофон</translation>
     </message>
 </context>
 </TS>
